.travis.yml

```diff
@@ -1,11 +1,8 @@
-language: php
-php:
-  - 5.4
-  - 5.5
-  - 5.6
+language: ruby
+
+services:
+  - docker
+
 before_script:
-  - composer self-update
-  - composer install
-  - cp "tests/env.travis" ".env"
-  - touch "storage/database.sqlite"
-  - php artisan migrate
\ No newline at end of file
+  - docker build savaslabs/squabble:latest .
+  - Make install
```

Comment:
We'll have to add in a `script:` block to run our phpunit tests in the container since we are no longer running the `language: php` box from Travis